lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_manager.h

```diff
@@ -41,6 +41,9 @@ constexpr char S1AP_TASK_NAME[]   = "S1AP";
 namespace magma {
 namespace lte {
 
+s1ap_state_t* create_s1ap_state(uint32_t max_enbs, uint32_t max_ues);
```

Comment:
Can you add documentation for this function, just to specify that this will create and return the S1AP task state struct?